lib/resources/google_apis.dart

```diff
@@ -0,0 +1,86 @@
+import 'dart:io' show Platform;
+import 'package:acs_upb_mobile/pages/timetable/model/events/uni_event.dart';
+import 'package:flutter/foundation.dart' show kIsWeb;
+import 'package:googleapis_auth/auth_io.dart';
+import 'package:googleapis/calendar/v3.dart';
+
+class GoogleApiHelper {
+  GoogleApiHelper();
+
+  static const List<String> _scopes = [CalendarApi.calendarScope];
+
+  static List<String> get scopes =>
```

Comment:
Here and below: move comments above what they refer to. Inline is usually used only for fairly short comments.